deploy/bz2067978/10-osd-delete-noderesolver-daemonsets.CronJob.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: osd-delete-noderesolver-daemonset-bz2067978
+  namespace: openshift-dns
+spec:
+  failedJobsHistoryLimit: 3
```

Comment:
Can we add a [`ttlSecondsAfterFinished`][1] too?  So that if we fail a few rounds of this but then the next few rounds pass, the failed runs will eventually be automatically pruned?  Because it seems unlikely that we want folks digging in to investigate and clear out any hiccup failures manually.

[1]: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#jobspec-v1-batch

Reply:
This would require moving to the batch/v1 version of the API. If we are targeting 4.8+ (k8s 1.21) then we can probably do that safely.

Reply:
Fair point! Now integrated.